dynatrace-operator-google-marketplace/chart/dynatrace-operator/templates/configmap.yaml

```diff
@@ -305,27 +305,23 @@ data:
     metadata:
       annotations:
         controller-gen.kubebuilder.io/version: v0.3.0
-      ownerReferences:
-      - apiVersion: v1beta1
-        blockOwnerDeletion: true
-        kind: Application
-        name: {{ .Release.Name }}
-        uid: ##UID##
```

Comment:
I presume you didn't want to delete this one.

Reply:
Re-added